config/default/crds/serving_v1alpha1_kfservice.yaml

```diff
@@ -144,6 +163,25 @@ spec:
                     0 in case of no traffic
                   format: int64
                   type: integer
+                pytorch:
+                  properties:
+                    modelClassFile:
+                      type: string
+                    modelClassName:
+                      description: Defaults to latest PyTorch Version.
+                      type: string
+                    modelUri:
+                      type: string
+                    resources:
+                      description: Defaults to requests and limits of 1CPU, 2Gb MEM.
+                      type: object
+                    runtimeVersion:
+                      type: string
+                  required:
+                  - modelUri
+                  - modelClassName
```

Comment:
isn't the plan for these to be optional (in which case a simple attempt to infer is made)?

Reply:
Done